docs/reference/capabilities.md

```diff
@@ -145,6 +145,7 @@ about capabilities, refer to the [Appium documentation](https://appium.io/docs/e
 |`appium:enablePerformanceLogging`| Enable Safari's performance logging (default `false`)| `true`, `false`|
 |`appium:autoWebview`| Move directly into Webview context if available. Default `false`|`true`, `false`|
 |`appium:skipTriggerInputEventAfterSendkeys`| If this capability is set to `true`, then whenever you call the Send Keys method in a web context, the driver will not fire an additional `input` event on the input field used for the call. This event, turned on by default, helps in situations where JS frameworks (like React) do not respond to the input events that occur by default when the underlying Selenium atom is executed. Default `false`|`true`, `false`|
+|`appium:sendKeyStrategy`| If this capability is set to `oneByOne`, then whenever you call the Send Keys method in a web context, the driver will type the given string one by one to the element. This strategy helps in situations where JS frameworks (like React) update the view for each input. If `appium:skipTriggerInputEventAfterSendkeys` capability is also set, it will affect every type. For example, when you send `appium` with `oneByOne` and `appium:skipTriggerInputEventAfterSendkeys` enables, the `appium:skipTriggerInputEventAfterSendkeys` affect each type, `a`, `p`, `p`,`i`, `u` and `m`. Otherwise the driver will type the given string at once to the element. `appium` Send Keys input types `appium` if `oneByOne` was not set. |`oneByOne`|
```

Comment:
will type each character the given string consists of in serial order

Reply:
> For example, when you send `appium` with `oneByOne` and `appium:skipTriggerInputEventAfterSendkeys` enables, the `appium:skipTriggerInputEventAfterSendkeys` affect each type

For example, when you are going to type the word `appium` using `oneByOne` strategy and `appium:skipTriggerInputEventAfterSendkeys` is enabled, the `appium:skipTriggerInputEventAfterSendkeys` option affects each typing action

Reply:
Otherwise the driver will type the given string as a whole in the target input element

Reply:
> Otherwise the driver will type the given string at once to the element.. `appium` Send Keys input types `appium` if `oneByOne` was not set.

If any other value or no value has been provided to the `appium:skipTriggerInputEventAfterSendkeys` capability then the driver types the given string as a whole in the destination input element.